doc/classes/Resource.xml

```diff
@@ -15,73 +17,85 @@
+				Override this method to customize the newly duplicated resource created from [method PackedScene.instance], if the original's [member resource_local_to_scene] is set to [code]true[/code].
+				[b]Example:[/b] Set a random [code]damage[/code] value to every local resource from an instantiated scene.
+				[codeblock]
+				extends Resource
+
+				var damage = 0
+
+				func _setup_local_to_scene():
+				    damage = randi_range(10, 40)
+				[/codeblock]
 			</description>
 		</method>
 		<method name="duplicate" qualifiers="const">
 			<return type="Resource" />
 			<argument index="0" name="subresources" type="bool" default="false" />
 			<description>
-				Duplicates the resource, returning a new resource with the exported members copied. [b]Note:[/b] To duplicate the resource the constructor is called without arguments. This method will error when the constructor doesn't have default values.
-				By default, sub-resources are shared between resource copies for efficiency. This can be changed by passing [code]true[/code] to the [code]subresources[/code] argument which will copy the subresources.
-				[b]Note:[/b] If [code]subresources[/code] is [code]true[/code], this method will only perform a shallow copy. Nested resources within subresources will not be duplicated and will still be shared.
-				[b]Note:[/b] When duplicating a resource, only [code]export[/code]ed properties are copied. Other properties will be set to their default value in the new resource.
+				Duplicates this resource, returning a new resource with its [code]export[/code]ed or [constant PROPERTY_USAGE_STORAGE] properties copied from the original.
+				If [code]subresources[/code] is [code]false[/code], a shallow copy is returned; nested resources within subresources are not duplicated and are shared from the original resource. If [code]subresources[/code] is [code]true[/code], a deep copy is returned; nested subresources will be duplicated and are not shared.
+				[b]Note:[/b] For custom resources, this method will fail if [method Object._init] has been defined with required parameters.
 			</description>
 		</method>
 		<method name="emit_changed">
 			<return type="void" />
 			<description>
-				Emits the [signal changed] signal.
-				If external objects which depend on this resource should be updated, this method must be called manually whenever the state of this resource has changed (such as modification of properties).
-				The method is equivalent to:
+				Emits the [signal changed] signal. This method is called automatically for some built-in resources.
+				[b]Note:[/b] For custom resources, it's recommended to call this method whenever a meaningful change occurs, such as a modified property. This ensures that custom [Object]s depending on the resource are properly updated.
 				[codeblock]
-				emit_signal("changed")
+				var damage:
+				    set(new_value):
```

Comment:
This syntax is 4.x exclusive, setget is required for 3.x AFAIK

Reply:
Oops, true.